tests/trainer/test_ppo_trainer.py

```diff
@@ -61,6 +67,37 @@ class PPOTrainerTester(unittest.TestCase):
     A wrapper class for testing PPOTrainer
     """
 
+    @classmethod
+    def setUpClass(cls):
+        cls._token = CI_HUB_USER_TOKEN
+        set_access_token(CI_HUB_USER_TOKEN)
+        HfFolder.save_token(CI_HUB_USER_TOKEN)
+        cls._api = HfApi(endpoint=CI_HUB_ENDPOINT)
+
+    @classmethod
+    def tearDownClass(cls):
+        for model in ["test-ppo-trainer"]:
+            try:
+                delete_repo(token=cls._token, repo_id=model)
+            except HTTPError:
+                pass
+
+    def setUp(self):
+
+        # model_id
+        model_id = "gpt2"
+
+        # get models and tokenizer
+        self.gpt2_model = AutoModelForCausalLMWithValueHead.from_pretrained(model_id)
```

Comment:
Do we need fresh models for each unit test? If not, I can move this to `setUpClass()` which is only called once (and saves time)

Reply:
Actually I don't think we need a fresh model for each test, yes can you move it to `setUpClass()` instead 🙏 Thanks!

Reply:
Done in https://github.com/lvwerra/trl/pull/68/commits/6c4ab687c2b9c24bee4fe9a56aa5877e1f8b896c